Library/Homebrew/extend/os/mac/dependency_collector.rb

```diff
@@ -8,7 +8,7 @@ class DependencyCollector
   def git_dep_if_needed(tags); end
 
   def subversion_dep_if_needed(tags)
-    Dependency.new("subversion", tags) if MacOS.version >= :catalina
+    Dependency.new("subversion", tags)
```

Comment:
👍🏻 on doing this sort of thing when the conditional logic applies to all our supported versions.

Reply:
Originally it was added because SVN is no longer provided with the CLT since Catalina, but now every CLT version of SVN from previous OSes has issues with some servers' certificates (e.g. svn.code.sf.net).

Reply:
Gotcha. Do we still need the `--trust-server-cert-failures=expired,not-yet-valid` then if we're going to unconditionally use our own Subversion?

Reply:
It's still needed if `trust_cert: true` is used on a URL.

Reply:
Gotcha. Is that cask-only?

Reply:
That's where it [originated from](https://github.com/Homebrew/homebrew-cask/pull/2822), and although it could be used in formulae, it doesn't appear in core currently. It could still be useful for casks until auto-install of SVN is implemented in cask/installer.rb#fetch.